src/Microsoft.DotNet.Core.Build.Tasks/GenerateRuntimeConfigurationFiles.cs

```diff
@@ -121,10 +119,24 @@ private void AddAdditionalProbingPaths(RuntimeOptions runtimeOptions)
                 }
 
                 // DotNetHost doesn't handle additional probing paths with a trailing slash
-                runtimeOptions.AdditionalProbingPaths.Add(PathUtility.EnsureNoTrailingSlash(packageFolder.Path));
+                runtimeOptions.AdditionalProbingPaths.Add(EnsureNoTrailingSlash(packageFolder.Path));
             }
         }
 
+        private static string EnsureNoTrailingSlash(string path)
+        {
+            if (!string.IsNullOrEmpty(path))
+            {
+                char lastChar = path[path.Length - 1];
+                if (lastChar == Path.DirectorySeparatorChar || lastChar == '/')
```

Comment:
doesn't it seem like a bug though?

---
In reply to: [73044608](https://github.com/dotnet/cli/pull/3962#discussion_r73044608) [](ancestors = 73044608)